### PR TITLE
Update to work with influxdb 0.9 - try 2

### PR DESCRIPTION
**Pull request by timgriffiths**
So this captures everything that is in pull request #29 but also handle all the special "characters" the influxdb protocol wants your to treat in a special way, ie spaces and commas and equal signs.

### CODE REVIEW

**Comment by contentfree** [comment]
@timgriffiths, You've got to sign the CLA. (I think you said you had in #29, but it doesn't appear so here)

**Comment by timgriffiths** [comment]
I really thought i did ... let me try again 
```
Created: 2/1/16
By: CLA Committers (committers@elasticsearch.com)
Status: SIGNED
Transaction ID: CBJCHBCAABAAEUxKCiz3OGog52g8P8osKhzO52D6mVUs
```

**Comment by timgriffiths** [comment]
Now doubly signed up
```
Created: 2/3/16
By: CLA Committers (committers@elasticsearch.com)
Status: SIGNED
Transaction ID: CBJCHBCAABAA6SjtIjPPAI_VAHThA0sINoFQ4njqOKNX
```

**Comment by suyograo** [comment]
Tests pass

```
suyog@machine:~/ws/elastic/ls_plugins/logstash-output-influxdb (pr/30)$ bundle exec rspec
Using Accessor#strict_set for specs
Run options: exclude {:redis=>true, :socket=>true, :performance=>true, :couchdb=>true, :elasticsearch=>true, :elasticsearch_secure=>true, :export_cypher=>true, :integration=>true, :windows=>true}
.........

Finished in 1.62 seconds (files took 1.37 seconds to load)
9 examples, 0 failures

Randomized with seed 25191
```

Also confirmed CLA
